coda/coda_mdstore/views.py

```diff
@@ -532,7 +532,7 @@ def bagHTML(request, identifier):
     )
 
 
-def bagURLList(request, identifier):
+def bagURLList(request, identifier, bagfiles=None):
```

Comment:
I propose we rename the kwarg something like `rendered` or `html`, and default it to `False` here and pass it as `True` in the new URL.

Reply:
I agree with Gio about a Boolean value here. I like `html` as the kwarg.